### ISSUE TITLE
visible focus indicator

### ISSUE THREAD

**Issue by karenhawkins** [PublicisSapient/accessible-ecommerce-demo#160]
Iris did not provide an Invision or sketch design of the visible focus indicator. But she did text me the requirements:
- colour: #079FFF
- outline: 2 px
- offset: 2 px

**Comment by aevanson** [comment]
@karenhawkins 
1) Does this colour work for both black and white backgrounds? 
2) Should it be a dotted line or a solid one?
3) In some cases, the offset might push it out of visibility (ie: scrolling cards in mini-cart). Might need to add margins around these items or tweak the offsets for these edge-cases.

**Comment by karenhawkins** [comment]
1. I don't know if Iris checked. Are you able to check? I am pretty sure it would. 
2. I think it should be solid, not dotted. 
3. Offset etc. is just so not in my world. Please do what you think is best, and Iris can confirm when she is back on Mon Apr 29. Hopefully that kool.

**Comment by aevanson** [comment]
1) Does it have to meet the "Graphical Objects and User Interface Components" test? If so, it will pass for a black background:
PASS: https://webaim.org/resources/contrastchecker/?fcolor=079FFF&bcolor=000000
But not a white one:
FAIL: https://webaim.org/resources/contrastchecker/?fcolor=079FFF&bcolor=FFFFFF

Tweaking the lightness _ever so slightly_ it's possible to get a pass with both background colours:
PASS: https://webaim.org/resources/contrastchecker/?fcolor=0097F5&bcolor=FFFFFF
PASS: https://webaim.org/resources/contrastchecker/?fcolor=0097F5&bcolor=000000

CC: @karenhawkins, @alisonrae 

Adding Alison because I'm not sure what test to use for outline contrast

**Comment by brianelton** [comment]
Fixed on public site
